frontend/src/routes/(authenticated)/project/[project_code]/+page.ts

```diff
@@ -74,6 +76,10 @@ export async function load(event: PageLoadEvent) {
 						flexProjectMetadata {
 							lexEntryCount
 						}
+            organizations {
```

Comment:
Whitespace is different from surrounding code, because your editor inserted spaces while the surrounding code was using tabs. Leave it as-is, we'll fix this in a different PR later on.